.eslintrc.js

```diff
@@ -7,6 +7,10 @@ module.exports = {
     '@metamask/eslint-config',
     '@metamask/eslint-config/config/nodejs',
   ],
+  rules: {
+    // TODO: Re-enable this post-TS migration
+    'import/no-unresolved': 'off',
+  },
```

Comment:
Just for my own understanding, why do we have to do this?

Reply:
ESLint should be able to run without the built files 🤔 but this comment doesn't make any sense

Reply:
I'll try and remove this, and add the a the prelint step run the build